### PR TITLE
[objc][docs] Fix relative links in getting-started-objective-c

### PR DESCRIPTION
**Pull request by VincentDondain**
Fix minor typos.

### CODE REVIEW

File: docs/getting-started-objective-c.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ Objective-C is a language that is most commonly used to write applications for m
 
 ### macOS
 
-[Creating a macOS application](getting-started-objc-macos) is easiest since it does not involve as many additional steps, like setting up identity, provisining profiles, simulators and devices. You are encouraged to start with the macOS document before the one for iOS.
+[Creating a macOS application](getting-started-objc-macos.md) is easiest since it does not involve as many additional steps, like setting up identity, provisining profiles, simulators and devices. You are encouraged to start with the macOS document before the one for iOS.
```
Comment (spouliot) [inline docs/getting-started-objective-c.md]: are you sure the `.md` extensions are needed ? iirc most of our documents (in master or in xamarin-macios wiki) don't use them.

Comment (VincentDondain) [inline docs/getting-started-objective-c.md]: Seems so. Without it the created link is https://github.com/VincentDondain/Embeddinator-4000/blob/fix-getting-started-objc/docs/Contributing#objective-c

Comment (VincentDondain) [inline docs/getting-started-objective-c.md]: The wiki (links to pages could be slightly different). When we reference `md` files from the repo we usually have absolute links (couldn't find a relative one without md).

**Comment by spouliot** [comment]
Travis hung (not related)

**Pull request merged**